web/src/classic/components/molecules/EarthEditor/PropertyPane/PropertyField/TypographyField/FontFamilyField/index.stories.tsx

```diff
@@ -15,16 +13,8 @@ export const Selected = () => (
   <FontFamilyField value="Comic Sans MS" onChange={action("onchange")} />
 );
 export const Linked = () => (
-  <FontFamilyField
-    value="Comic Sans MS"
-    color={darkColors.primary.main}
-    onChange={action("onchange")}
-  />
+  <FontFamilyField value="Comic Sans MS" color={"#00A0E8"} onChange={action("onchange")} />
```

Comment:
classic code shouldn't be affected by any changes in this PR. I guess when I split the themes again I missed this import. 
Please find the corresponding color in classic's theme and use that instead 🙏🏻